### PR TITLE
🎓 Lab objects redo lab

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
Redo the objects/class lab. 

### Why
Student feedback from the subsequent course (CS102) was that they wanted more practice with objects/classes in CS101. This lab is a more basic version of what it was, and there will be another objects lab following this one that will be a little more fancy. The following lab will be a collection of some sort I think.

### CODE REVIEW

**Comment by jameshughes89** [comment]
There is a clear emphasis on geometry, but I figure the simple escalation in complexity will help with them generalizing.

File: site/labs/objects/objects.rst [hunk 1]
```diff
@@ -2,194 +2,91 @@
...
-* A first name (``_first_name``)
-* A last name (``_last_name``)
-* An email (``_email``)
+    You must have completed the specified exercises prior to the start of the lab. If you have not come to lab prepared,
+    you will be asked to leave and you will obtain a grade of 0 for the lab.
 
-There, that's it for now. We can add more later if we want. 
 
-**Methods**
+* Have a working implementation of the ``Circle`` class from :doc:`lecture </topics/objects/objects1>`
```
Comment (twentylemon) [inline site/labs/objects/objects.rst:31]: They'd get a 0 if they don't copy/paste a file from the notes? Seems a little fussy.

File: site/labs/objects/objects.rst [hunk 2]
```diff
@@ -2,194 +2,91 @@
...
-5. https://open.kattis.com/problems/modulo
+#. Create a ``Person`` class
 
-LeetCode Problems
-=================
+    * Attribute for ``first_name``
+    * Attribute for ``last_name``
+    * Attribute for ``email_address``
+    * The constructor will take three parameters for specifying the values of the attributes
+    * A method returning a string of their ``full_name()``
```
Comment (twentylemon) [inline site/labs/objects/objects.rst:70]: There's lots of definitions of _full name_. Maybe worth disambiguating here and saying that any combination of their first/last names is sufficient?

**Review by twentylemon — APPROVED** [review]

File: site/labs/objects/objects.rst (hunk 1 above)
Comment (jameshughes89) [reply site/labs/objects/objects.rst:31]: It's to ensure they have it ready to go as a template. I will add a note like _yes, it's a simple pre-lab exercise this week_.

Comment (twentylemon) [reply site/labs/objects/objects.rst:31]: Still. To give 0 here seems excessive. It'll take like literally 10 seconds. You could even add a link to download the file.

Comment (jameshughes89) [reply site/labs/objects/objects.rst:25]: I feel ya. 
1. Removed warning about 0
2. Added download link

**Review by twentylemon — APPROVED** [review]

**Pull request merged**